src/SparseDiffTools.jl

```diff
@@ -1,7 +1,8 @@
 module SparseDiffTools
 
 using SparseArrays, LinearAlgebra, BandedMatrices, BlockBandedMatrices,
-      LightGraphs, VertexSafeGraphs, DiffEqDiffTools, ForwardDiff, Zygote
+      LightGraphs, VertexSafeGraphs, DiffEqDiffTools, ForwardDiff, Zygote,
```

Comment:
that makes a lot of packages required, could some of them be `import`-ed to simplify reading (or `using` only the identifiers needed)